include/circt/Dialect/ESI/cosim/CosimDpi.capnp

```diff
@@ -0,0 +1,35 @@
+@0xe642127a31681ef6;
```

Comment:
Please add something like the standard file header (with the right comment marker of course) to all new files.